packages/element/src/create-interpolate-element.js

```diff
@@ -22,28 +24,44 @@ let indoc, offset, output, stack;
  */
 const tokenizer = /<(\/)?(\w+)\s*(\/)?>/g;
 
+/**
+ * @typedef Frame
```

Comment:
We could include a brief description of the type.

```suggestion
/**
 * The stack frame tracking parse progress.
 *
 * @typedef Frame
```

It caught me off guard reading through this, since it was not at all clear to me at first glance what a "frame" was.